### ISSUE TITLE
Build fails when hidden files are in the styles directory

### ISSUE THREAD

**Issue by max** [salsify/ember-css-modules#21]
If you have a hidden file like `.DS_Store` or `.gitkeep` in `app/styles` the build fails with the following output:

```
The Broccoli Plugin: [BroccoliMergeTrees] failed with:
Error: Merge error: file hidden-files-css-modules/styles/.gitkeep exists in /Users/my-user/Code/hidden-files-css-modules/tmp/broccoli_merge_trees-input_base_path-waG9v3er.tmp/0 and /Users/my-user/Code/hidden-files-css-modules/tmp/broccoli_merge_trees-input_base_path-waG9v3er.tmp/1
Pass option { overwrite: true } to mergeTrees in order to have the latter file win.
    at BroccoliMergeTrees._mergeRelativePath (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/index.js:212:17)
    at BroccoliMergeTrees._mergeRelativePath (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/index.js:238:31)
    at BroccoliMergeTrees._mergeRelativePath (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/index.js:238:31)
    at BroccoliMergeTrees.build (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/index.js:50:24)
    at /Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/read_compat.js:61:34
    at lib$rsvp$$internal$$tryCatch (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:1019:11)
    at lib$rsvp$asap$$flush (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:1198:9)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)

The broccoli plugin was instantiated at:
    at BroccoliMergeTrees.Plugin (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/node_modules/broccoli-plugin/index.js:7:31)
    at new BroccoliMergeTrees (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/node_modules/broccoli-merge-trees/index.js:29:10)
    at ModulesPreprocessor.inputTreeWithStyles (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/lib/modules-preprocessor.js:62:10)
    at ModulesPreprocessor.toTree (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-css-modules/lib/modules-preprocessor.js:34:30)
    at /Users/my-user/Code/hidden-files-css-modules/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:184:26
    at Array.forEach (native)
    at processPlugins (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:182:11)
    at module.exports.preprocessJs (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:175:10)
    at EmberApp.appAndDependencies (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-cli/lib/broccoli/ember-app.js:1044:25)
    at EmberApp.javascript (/Users/my-user/Code/hidden-files-css-modules/node_modules/ember-cli/lib/broccoli/ember-app.js:1159:34)
```

**Comment by dfreeman** [comment]
Not positive, but it looks like when Ember CLI filters out the styles directory from the app tree, dotfiles are actually being left behind, so then when we merge the styles directory back in, they're colliding.
I'll have to see if I can play any tricks with `broccoli-funnel` to avoid this.

**Comment by dfreeman** [comment]
Just released 0.3.2 with a fix for this
